src/need-to-know.sql

```diff
@@ -296,26 +269,19 @@ $$ language plpgsql;
 revoke all privileges on function ntk.sql_type_from_generic_type(text) from public;
 grant execute on function ntk.sql_type_from_generic_type(text) to admin_user;
 
+drop type if exists access_control_type;
+create type access_control_type as enum('mac', 'generic');
 
 drop function if exists table_create(json, text, int);
-create or replace function table_create(definition json, type text, form_id int default 0)
+create or replace function table_create(untrusted_definition json, untrusted_type access_control_type, form_id int default 0)
     returns text as $$
-    declare untrusted_definition json;
-    declare untrusted_type text;
-    declare untrusted_form_id int;
-    declare _res text;
     begin
-        untrusted_definition := definition;
-        untrusted_type := type;
-        untrusted_form_id := form_id;
         if untrusted_type = 'mac' then
-            select ntk.parse_mac_table_def(untrusted_definition) into _res;
-            return _res;
+            select ntk.parse_mac_table_def(untrusted_definition);
```

Comment:
I get this error here, when running the tests: `query has no destination for result data
HINT:  If you want to discard the results of a SELECT, use PERFORM instead.`.